app/cdap/components/shared/ConfirmDialog/index.tsx

```diff
@@ -23,8 +23,20 @@ import KeyboardArrowDownIcon from '@material-ui/icons/KeyboardArrowDown';
 import KeyboardArrowUpIcon from '@material-ui/icons/KeyboardArrowUp';
 
 import IconButton from '@material-ui/core/IconButton';
-import { StyledBox, StyledDialog, StyledAlert } from './styles';
+import { StyledBox, StyledDialog, StyledAlert, CopyContentBox } from './styles';
 import PrimaryTextButton from 'components/shared/Buttons/PrimaryTextButton';
+import { Color } from '@material-ui/lab/Alert';
+import { copyToClipBoard } from 'services/Clipboard';
+import FileCopyOutlinedIcon from '@material-ui/icons/FileCopyOutlined';
+
+export type Severity = Color;
+
+export enum SeverityType {
```

Comment:
where is this used?

Reply:
This would be used by consumer components of this Confirm dialog. Something like this:
```
<ConfirmDialog
      headerTitle={'Delete'}
     ....
      cancelButtonText={T.translate(`${PREFIX}.cancelButtonText`)}
      confirmButtonText={T.translate(`${PREFIX}.deleteButtonText`)}
     .....
      severity={SeverityType.ERROR}
      statusMessage={deleteErrorMsg}  >
</ConfirmDialog>
```

Reply:
But the `severity` field takes type `Severity` not `SeverityType`?

Reply:
Streamlined the code now.